projects/ngx-charts-on-fhir/src/lib/fhir-chart/range-selector/range-selector.component.ts

```diff
@@ -69,7 +69,14 @@ export class RangeSelectorComponent {
       })
       let sortedData: any[] = [];
       for (let item of data) {
-        let xcordinates = item.map((el: any) => el.x)
+        let xcordinates = item.map(function (el: any) {
+          if (!isNaN(el.x)) {
+            return el.x;
+          }
+        })
+        xcordinates = xcordinates.filter(function( element:any ) {
+          return element !== undefined;
+       });
```

Comment:
this could be simplified by just filtering out NaN instead of converting it to undefined first